packages/terra-legacy-theme/README.md

```diff
@@ -3,29 +3,114 @@
+- Install from [npmjs](https://www.npmjs.com): `npm install terra-legacy-theme`
 
-## How to replace this package?
-Instead of using this file, we've opted to hard-code these values as fallback values directly into the source code of our components using the CSS custom property syntax. This has eliminated the need for using this package in the terra-ui components. This approach may be applicable for teams as they move away from using this package as-well.
+## Usage
+This project offers two formats of terra-legacy-theme based around [browsers support for native CSS custom properties](http://caniuse.com/#search=custom%20properties) .
 
-~~The legacy-theme component sets global variables for the entire application. Global variables are included for responsive breakpoints, colors, and typography.~~
+* One that can be used in browsers that support native support for CSS custom properties as well as browsers that lack support for native CSS custom properties.
+* One that can only be used in browsers that support native support for CSS custom properties.
 
-- [Getting Started](#getting-started)
-- [Documentation](https://github.com/cerner/terra-core/tree/master/packages/terra-legacy-theme/docs)
-- [LICENSE](#license)
+It is recommended to use the first format when using terra-legacy-theme as your default theme.
 
-## Getting Started
+### Format 1 - Root Theme
+This theme format can be used to replace terra's default theme variables with custom theme variables and can be compiled turning the themed CSS custom properties into static values. The static values work in both browsers that lack support for native CSS custom properties and browsers that support CSS custom properties.
 
-- Install from [npmjs](https://www.npmjs.com): `npm install terra-legacy-theme`
+In your application, import the `terra-legacy-theme` module like so:
+
+```js
+// ES6 import
+import 'terra-legacy-theme';
+```
+
+#### Format 1 Setup
+
+You'll need to set up some PostCSS tools to ensure the terra-legacy-theme can be compiled correctly.
+
+* [postcss-assets-webpack-plugin](https://github.com/klimashkin/postcss-assets-webpack-plugin)
+* [postcss-custom-properties](https://github.com/postcss/postcss-custom-properties)
+
+`npm install postcss-assets-webpack-plugin --save-dev`
+
+`npm install postcss-custom-properties --save-dev`
+
+The postcss-assets-webpack-plugin gets css, extracted by [ExtractTextPlugin](https://github.com/webpack/extract-text-webpack-plugin) and applies postcss plugins to it.
```

Comment:
We should point out that `extract-text-webpack-plugin` doesn't have webpack 4 support. If consumers use their own webpack config utilizing webpack v4, they should use `mini-css-extract-plugin`.